src/detail/sqlite_backend.cc

```diff
@@ -76,16 +137,58 @@ struct sqlite_backend::impl {
     sqlite3_close(db);
   }
 
-  bool open(const std::string& path) {
-    BROKER_TRACE(BROKER_ARG(path));
+  // Run PRAGAMA command with an optional value.
+  //
+  // Assumes name and value have been verified previously and are safe
+  // to be formatted into SQL.
+  bool exec_pragma(const std::string& name,
+                   const std::string& value = std::string()) {
+    auto query = std::string("PRAGMA ");
```

Comment:
Please follow Broker's coding style and general best practice (https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es23-prefer-the--initializer-syntax).

```suggestion
    auto query = std::string{"PRAGMA "};
```